src/components/SQForm/SQFormButton.tsx

```diff
@@ -3,34 +3,41 @@ import {RoundedButton} from 'scplus-shared-components';
 import {useFormButton, BUTTON_TYPES} from './useFormButton';
 import type {ButtonType} from './useFormButton';
 
-export type SQFormButtonProps = {
+export type SQFormButtonProps<Values = void> = {
```

Comment:
I think we need to use something like `unknown` for this default value. `void` is for functions, but you're using Values as a type for an object.

Reply:
@laurelbean Sounds good.  I'll look into updating this.